[2020-06-12 14:36:09,504] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: dataops.modeling 2020-06-11T00:00:00+00:00 [queued]>
[2020-06-12 14:36:09,533] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: dataops.modeling 2020-06-11T00:00:00+00:00 [queued]>
[2020-06-12 14:36:09,533] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-12 14:36:09,533] {taskinstance.py:880} INFO - Starting attempt 7 of 10
[2020-06-12 14:36:09,533] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-12 14:36:09,554] {taskinstance.py:900} INFO - Executing <Task(BashOperator): modeling> on 2020-06-11T00:00:00+00:00
[2020-06-12 14:36:09,558] {standard_task_runner.py:53} INFO - Started process 21751 to run task
[2020-06-12 14:36:09,624] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: dataops.modeling 2020-06-11T00:00:00+00:00 [running]> vinicius-Latitude-3480
[2020-06-12 14:36:09,649] {bash_operator.py:82} INFO - Tmp dir root location: 
 /tmp
[2020-06-12 14:36:09,650] {bash_operator.py:103} INFO - Temporary script location: /tmp/airflowtmpcx539xef/modeling29ag86nl
[2020-06-12 14:36:09,650] {bash_operator.py:115} INFO - Running command: python /home/vinicius/Documents/Projetos/Claro/Piloto/git_Homologação/src/modeling/run.py
[2020-06-12 14:36:09,656] {bash_operator.py:122} INFO - Output:
[2020-06-12 14:37:39,990] {bash_operator.py:126} INFO - Treinando modelo KN_Test
[2020-06-12 14:37:39,990] {bash_operator.py:126} INFO - INFO: 'Test_model' does not exist. Creating a new experiment
[2020-06-12 14:37:39,990] {bash_operator.py:126} INFO - Treinando modelo KN_4
[2020-06-12 14:37:39,990] {bash_operator.py:126} INFO - Treinando modelo test_LR
[2020-06-12 14:37:39,990] {bash_operator.py:126} INFO - Tabela salva em /home/vinicius/Documents/Projetos/Claro/Piloto/git_Homologação/src/modeling/output/model_output_83459.csv
[2020-06-12 14:37:39,990] {bash_operator.py:126} INFO - dados dos modelos salvos em /home/vinicius/Documents/Projetos/Claro/Piloto/git_Homologação/src/modeling/output/model_output_83459.csv
[2020-06-12 14:37:40,294] {bash_operator.py:128} INFO - Command exited with return code 0
[2020-06-12 14:37:40,327] {taskinstance.py:1052} INFO - Marking task as SUCCESS.dag_id=dataops, task_id=modeling, execution_date=20200611T000000, start_date=20200612T173609, end_date=20200612T173740
[2020-06-12 14:37:45,154] {logging_mixin.py:112} INFO - [2020-06-12 14:37:45,154] {local_task_job.py:103} INFO - Task exited with return code 0
